pls_balances/scripts/generate.py

```diff
@@ -108,6 +108,8 @@ def compositional_effect_size(max_alpha, reps, intervals,
               help='Number of effect size benchmarks to test.')
 @click.option('--n-species', default=100,
               help='Number of species')
+@click.option('--asymmetry', default=False,
```

Comment:
We will want to add in `is_flag=True` http://click.pocoo.org/5/options/#boolean-flags